pkg/monitor/monitor.go

```diff
@@ -41,21 +43,27 @@ type controllerMonitor struct {
 	client       clientset.Interface
 	nodeInformer coreinformers.NodeInformer
 	// nodeListerSynced is a function which returns true if the node shared informer has been synced at least once.
-	nodeListerSynced cache.InformerSynced
+	nodeListerSynced    cache.InformerSynced
+	addressGroupStore   storage.Interface
+	appliedToGroupStore storage.Interface
+	networkPolicyStore  storage.Interface
```

Comment:
How about doing the same for controller side networkpolicyinfo querier so that others can use this querier too?

Reply:
Sounds good, thanks. Will update accordingly.